x-pack/plugins/uptime/public/components/monitor_management/test_now_mode/browser/browser_test_results.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as React from 'react';
+import { EuiAccordion, EuiText } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import styled from 'styled-components';
+import { StepsList } from '../../../synthetics/check_steps/steps_list';
+import { JourneyStep } from '../../../../../common/runtime_types';
+import { useBrowserRunOnceMonitors } from './use_browser_run_once_monitors';
+import { TestResultHeader } from '../test_result_header';
+
+interface Props {
+  monitorId: string;
+}
+export const BrowserTestRunResult = ({ monitorId }: Props) => {
+  const { data, loading, stepEnds, journeyStarted, summaryDoc, stepListData } =
+    useBrowserRunOnceMonitors({
+      monitorId,
+    });
+
+  const hits = data?.hits.hits;
+  const doc = hits?.[0]?._source as JourneyStep;
+
+  const buttonContent = (
+    <div>
+      <TestResultHeader
+        summaryDocs={summaryDoc ? [summaryDoc] : []}
+        doc={doc}
+        journeyStarted={journeyStarted}
+        isCompleted={Boolean(summaryDoc)}
+      />
+      <EuiText size="s">
+        <p>
+          <EuiText color="subdued">
+            {i18n.translate('xpack.uptime.monitorManagement.stepCompleted', {
+              defaultMessage: '{noOfSteps} {stepLabel} completed',
```

Comment:
[This](https://github.com/elastic/kibana/blob/d40c0abe5bc1d0a1c3514068603f0f9a0dffc7a7/packages/kbn-i18n/README.md?plain=1#L241) format can also be used for proper pluralization.

Also, number 0 is a plural. i.e. `"0 steps completed"`.

![image](https://user-images.githubusercontent.com/2748376/150173464-b6add915-59eb-4f28-b1c6-8d3988bced26.png)